src/TpdaLiquidationPair.sol

```diff
@@ -12,7 +12,7 @@ import { IFlashSwapCallback } from "pt-v5-liquidator-interfaces/IFlashSwapCallba
 error SwapExceedsMax(uint256 amountInMax, uint256 amountIn);
 
 /// @notice Thrown when there is zero available balance to swap
-error ZeroAvailableBalance();
+error InsufficientBalance();
```

Comment:
```suggestion
/// @param amountOut The amount out requested
/// @param availableBalance The available balance than can be transferred out
error InsufficientBalance(uint256 amountOut, uint256 availableBalance);
```

We can give some more context in this error on the state that led to it being thrown.